src/DataDefinitionsBundle/Resources/public/pimcore/js/export/search.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Data Definitions.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2016-2019 w-vision AG (https://www.w-vision.ch)
+ * @license    https://github.com/w-vision/ImportDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ */
+
+pimcore.object.search = Class.create(pimcore.object.search, {
```

Comment:
not sure if this might cause issues if other plugins also overwrite it....

Reply:
Do you have any alternative suggestions?

Reply:
isn't there any other extension point available?

Reply:
Doesn't look like it, sadly. I've looked and only answers to that question were yours, suggesting to do exactly this. :rofl:

Reply:
yes, I am happy to do that within a project, but I am not sure within a bundle....

So, since it's currently the only way, we should go for it...

Reply:
https://talk.pimcore.org/t/plugin-js-question/278/5

Reply:
Yes, I fully agree, if there was another way, I would have used it. Maybe Pimcore could fire an event after creating the search grid? But that would only be from now on, 6.x.